code/modules/research/techweb/_techweb_node.dm

```diff
@@ -87,6 +109,10 @@
 			for(var/i in L)
 				if(actual_costs[i])
 					actual_costs[i] -= L[i]
+		for(var/i in actual_costs)
```

Comment:
```suggestion
		for(var/cost in actual_costs)
```
This line does not meet our contribution standards, specifically for [variable name compliance](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names), and should be refactored.